regolith/helpers/l_membershelper.py

```diff
@@ -23,6 +23,7 @@
 def subparser(subpi):
     subpi.add_argument("-v", "--verbose", action="store_true", help='increase verbosity of output')
     subpi.add_argument("-c", "--current", action="store_true", help='get only current group members ')
+    subpi.add_argument("-f", "--former", action="store_true", help='most recent employment of former members ')
```

Comment:
the help is more useful if it tells what the thing does, so this should say  sth like ``"get only former group members"`` and then it should do that.

But someone is "former" if they are in people.yml but are ``active: False``

I think it is helpful if we also print in verbose the current position (if they are current) or the position they had in the group (if they are former) so we can distinguish if they were a short-term visitor or a grad student, for example.